src/views/Main.tsx

```diff
@@ -0,0 +1,44 @@
+import {Container, Title, ChatButton, ChatWrapper,ButtonWrapper} from '../styles/style.main';
+import Chat from '../components/Chat';
+import { useRecoilState } from 'recoil';
+import {TbBrandHipchat} from 'react-icons/tb';
+import {showDivState, hideButtonState} from '../store/atom';
+
+
+function Main(): JSX.Element {
+  const [showDiv, setShowDiv] = useRecoilState(showDivState);
+  const [hideButton, setHideButton] = useRecoilState(hideButtonState);
+  const handleContextMenuClick = (e : any) => {
+    e.preventDefault();
+  }
+  const handleButtonClick = () => {
+    setShowDiv(true);
+    setHideButton(true);
+  };
+
+  return (
+    <>
+      <Container onContextMenu = {handleContextMenuClick}>
+        <ButtonWrapper>
+          {!hideButton &&

```

Comment:
채팅룸 들어가는 버튼을 숨기는 hideButton과 채팅룸을 나타나게하는 showDiv는 동작원리가 같아보여서, showDiv state로만 처리할 수 있을 것 같은데 혹시 따로 hideButton state를 만드신 이유가 있으신가요??
```suggestion
          {!showDiv &&
```